.changes/next-release/bugfix-s3 sync-86921.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We already included this in a release of v2. We should remove the changelog entry.